core/src/main/java/com/digitalpebble/stormcrawler/protocol/httpclient/HttpProtocol.java

```diff
@@ -112,6 +113,30 @@ public void configure(final Config conf) {
             defaultHeaders.add(new BasicHeader("Accept", accept));
         }
 
+        boolean useBasicAuth = ConfUtils.getBoolean(conf,
```

Comment:
Instead of having a separate configuration for activating or deactivating the authentication, what about checking if http.basicauth.user is not null? If it is set, we use it. That would keep things simpler.